staff/fatima-mata/precourse/bingo/bingo.js

```diff
@@ -0,0 +1,126 @@
+var bingoCard = [
```

Comment:
you play always with the same card. Can be set it with a random card each game?

Reply:
I had the wrong file. I have already uploaded the correct one.